docs/pin_mapping.md

```diff
@@ -1,22 +1,25 @@
 # Pin mapping
 
+![TPM1a](/docs/TPM1a.png)
+
 APU3/4 | TPM1a
 -----|-----
-1 | 11
-2 | 9
-3 | 7
-4 | 5
-5 | 3
-6 | GND
-7 | 13
-8 | 1
-9 | VCC
+1 (**LFRAME**) | 11
+2 (**LAD3**) | 9
+3 (**LAD2**) | 7
+4 (**LAD1**) | 5
+5 (**LAD0**) | 3
+6 (**gnd**) | 2, 4, 6, 8, 10, 12, 17
+7 (**LPCRST**) | 13
+8 (**LPCCLK1**) | 1
+9 (**V3**) | 18
 
 ## TPM1a
-Pins no. **2, 4, 6, 8, 10, 12 and 17** should be connected to **GND** and pin no. **18** should be connected to **V3**. Pins no. **14, 15, 16, 19 and 20** stay unconnected.
++ At least one pin of no. **2, 4, 6, 8, 10, 12 and 17** should be connected to **GND** and 
+pin no. **18** should be connected to **V3**.
++ Pins no. **14, 15, 16, 19 and 20** stay unconnected.
 
 
 ## APU3/4
-Pin no. **6** should be connected to **GND** and pin no. **9** should be connected to **V3**.
-
-![TPM1a](/docs/TPM1a.png)
++ Pin no. **6** should be connected to **GND** and pin no. **9** should be 
+connected to **V3**.
```

Comment:
@finus16 there is not note about the size of holes on LPC debug an apu3/4. This hsould be noted that normal pin header will not fit there